people/mone/page.en.md

```diff
@@ -0,0 +1,55 @@
+---
+name: mone
+info:
+    alias: 彤彤
+    location: Shenzhen, Guangdong
+---
+
+## Description
+
+Mone is one of cisgender allies among us, a very cute little girl.
+
+She loves idle groups and singing programmes in Japan and South Korea, and often reposts tweets about them.
+
+She also plays Minecraft. She built a small villa with a cozy room belongs to herself in game.
```

Comment:
She also plays Minecraft. In that game, she built a small villa with a cozy room belonging to herself.